### ISSUE TITLE
File relative values

### ISSUE THREAD

**Issue by marcospassos** [rollup/rollup-plugin-replace#1]
Hey folks, thanks for sharing this great plugin with us!

We've an use case where we need to replace some markups using file relative values. Perhaps, providing some constants would be a great addition to this plugin.

Example:

| Constant | Get replaced by |
| --- | --- |
| `__FILE__` | The current file path |
| `__FILENAME__` | The current file name |
| `__BASEDIR__` | The current base directory |

``` js
{
  values: {
    PATH: '__FILE__',
    VERSION: '1.0.0',
    ENVIRONMENT: JSON.stringify( 'development' )
  }
}
```

Thanks in advance!

**Comment by Rich-Harris** [comment]
@marcospassos sorry for the long delay. I've just opened #2 which tackles the problem from a different direction, by allowing values to be functions (that take a module ID). Would that meet your use case?

**Comment by Rich-Harris** [comment]
Fixed in 1.2
